JobConfig/cosmic/S1DSStops.fcl

```diff
@@ -6,8 +6,8 @@
 
 process_name : Primary
 
-physics.TriggerPath: [generate, genCounter, g4run, g4status, stepPointMomentumFilter, cosmicFilter]
-outputs.Output.outputCommands : [@sequence::Primary.KeptProducts, "keep mu2e::GenParticles_*_*_*", "keep *_cosmicFilter_*_*"]
+physics.TriggerPath: [generate, genCounter, g4run, g4status, stepPointMomentumFilter, cosmicFilter, compressPV]
```

Comment:
I don't see where the compressPV module is defined.  It's not in this file or any of the #includes.

Reply:
It's in JobConfig/primary/prolog.fcl (but defined in JobConfig/cosmic/prolog.fcl) no?